src/main/features/claim/form/models/eligibility/eligibility.ts

```diff
@@ -33,20 +36,23 @@ export class Eligibility {
   @IsIn(YesNoOption.all(), { message: ValidationErrors.YES_NO_REQUIRED, groups: [ValidationGroups.HELP_WITH_FEES] })
   helpWithFees?: YesNoOption
 
-  @IsIn(YesNoOption.all(), { message: ValidationErrors.YES_NO_REQUIRED, groups: [ValidationGroups.CLAIM_IS_FOR_TENANCY_DEPOSIT] })
```

Comment:
It might be a good idea to keep unrelated changes, especially ones that don't bring considerable improvements, out of the PR. In the long run, it would make it much easier to track down *why* something in the code has been changed. It's not a dealbreaker, but at least consider the reasons.

Reply:
That was done automatically by intellJ. The previous dev just didn't auto format the lines so when i did i changed their allignment